types/wlr_output_damage.c

```diff
@@ -31,6 +31,12 @@ static void output_handle_scale(struct wl_listener *listener, void *data) {
 	wlr_output_damage_add_whole(output_damage);
 }
 
+static void output_handle_subpixel(struct wl_listener *listener, void *data) {
+	struct wlr_output_damage *output_damage =
+		wl_container_of(listener, output_damage, output_subpixel);
+	wlr_output_damage_add_whole(output_damage);
```

Comment:
I don't think this is necessary.

Reply:
I damaged the whole screen to get commands like `swaymsg output eDP-1 subpixel rgb` to update the hinting immediately. Without this I'd have to resize/quit/move apps for the subpixel hinting to update in title bars.

Reply:
I see. This should be in sway then, because wlroots compositors might not need that. For instance rootston doesn't need to re-draw anything when subpixel changes.